### DISCUSSION TITLE
How to get start/end indices of hlgroups used in statusline?

### DISCUSSION THREAD

**Discussion by siduck** [neovim/neovim#31671] [Q&A] [answered]
I used nvim_eval_statusline but the last item's start index, was more than the statusline's width itself  .

So i can know the start/end indices of all hlgroups in statusline except for the last one

![image](https://github.com/user-attachments/assets/17300a31-4d8d-4a99-a019-c400f97f6ffe)

**Comment by zeertzjq** [comment, marked as answer]
Use the length of `str`
